ihatemoney/static/js/ihatemoney.js

```diff
@@ -1,18 +1,7 @@
- // Add scripts to select all or non of the checkboxes in the add_bill form
-function selectall()
-    {
-        var els = document.getElementsByName('payed_for');
-        for(var i =0;i<els.length;i++)
-        {
-            els[i].checked=true;
-        }
-    }
-function selectnone()
-    {
-        var els = document.getElementsByName('payed_for');
-        for(var i =0;i<els.length;i++)
-        {
-            els[i].checked=false;
-        }
-    }
-
+ // Utility to select all or none of the checkboxes in the add_bill form.
+function selectCheckboxes(value){
+  var els = document.getElementsByName('payed_for');
+  for(var i = 0; i < els.length; i++){
+    els[i].checked = value;
```

Comment:
if you can use es6: `for (let node of document.getElementsByName('payed_for')) { node.checked = value; }` for even more conciseness

Reply:
I'm not really sure which JS version / browsers we're targeting, so I'll let as is until we make a decision on this ;)